compiler/runtime/OMRCodeCacheManager.cpp

```diff
@@ -1075,8 +1082,8 @@ OMR::CodeCacheManager::carveCodeCacheSpaceFromRepository(size_t segmentSize,
          TR_VerboseLog::writeLineLocked(TR_Vlog_CODECACHE, "carved size=%u range: " POINTER_PRINTF_FORMAT "-" POINTER_PRINTF_FORMAT,
                   codeCacheSizeToAllocate, start, end);
       else
-         TR_VerboseLog::writeLineLocked(TR_Vlog_FAILURE, "failed to carve size=%lu. Free space = %u",
-                  codeCacheSizeToAllocate, freeSpace);
+         TR_VerboseLog::writeLineLocked(TR_Vlog_FAILURE, "failed to carve size=%lu. Free space = %u. isSufficientPhysicalMemoryAvailableForAllocation = %s",
```

Comment:
I would like the failure message to be printed when `verbose={compilePerformance}` is specified as well

Reply:
Sorry for not being more specific: the failure message is the one that is important for performance. The fact that we successfully carved a block of memory from the repository is an uninteresting event that should only be printed if verbose={codeCache} is specified.

Reply:
Another suggestion: in case of a failure the message is something like:
`#FAILURE:  failed to carve size=524288. Free space = 11010048. isSuficientPhysicalMemoryAvailableForAllocation = false`
It's not clear what we want to carve because there is no reference to code cache. I would change the message to something like:
`TR_VerboseLog::writeLineLocked(TR_Vlog_FAILURE, "failed to carve code cache of size=%zu from the repository. Free space in code cache repository = %zu. isSufficientPhysicalMemoryAvailableForAllocation = %s",`